queries.sql

```diff
@@ -55,7 +55,7 @@ BEGIN;
 DELETE FROM animals;
 ROLLBACK;
 
-
+/* TRANSACTION 2 */
```

Comment:
- [ ] In your last query /* What is the average number of escape attempts per animal type of those born between 1990 and 2000? */  
Kindly consider correcting the ending date which is `2000/12/31`.

Reply:
This has been resolved